pages/integrations/docker_hub.md.erb

```diff
@@ -0,0 +1,37 @@
+# Docker Hub
+
+[Docker Hub](https://hub.docker.com/) is a public registry of docker images, with popular images used in many build pipelines. 
+
+On 1st November 2020, [Docker Hub introduced strict rate limits](https://docs.docker.com/docker-hub/download-rate-limit/) on image downloads by unauthenticated clients and authenticated clients on a free plan. For many Buildkite customers, this may result in some jobs on intermittantly failing.
```

Comment:
some typo action

```suggestion
On 1st November 2020, [Docker Hub introduced strict rate limits](https://docs.docker.com/docker-hub/download-rate-limit/) on image downloads by unauthenticated clients and authenticated clients on a free plan. For many Buildkite customers, this may result in some jobs intermittently failing.
```